scripts/globals/mobskills/Medusa_Javelin.lua

```diff
@@ -16,16 +16,23 @@ end;
 
 function onMobWeaponSkill(target, mob, skill)
 
+	local mobID = mob:getID();
 	local numhits = 1;
 	local accmod = 1;
 	local dmgmod = 1.5;
+	if (mobID == 16912876) then 

```

Comment:
Like I was just telling someone else, static mobID checks aren't such a great idea.
https://github.com/DarkstarProject/darkstar/pull/2471/files#diff-b0e2bde3c212fb2418d5e7a6df1f62e9R14
We can actually check the skill list via a mobMod though. `mob:getMobMod(MOBMOD_SKILL_LIST) == LIST_ID_HERE`